calcom/cal.com
# fix: fixes #11558 : Added key to the props for Cal.tsx

/claim #11558 

## What does this PR do?

This PR fixes #11558 by adding a key value to the props passed to the Cal iframe by any external URL.

## Type of change



- [ x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
This can be tested by passing the key prop any random value, or the src url. This should result in the page being reloaded with the iframe and not having to click the back button twice in the browser.

- [ x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.
